app/index.js

```diff
@@ -75,6 +75,17 @@ const saveIfAllCollected = () => {
 }
 
 app.on('ready', function () {
+  app.on('certificate-error', function (e, webContents, url, error, cert, cb) {
+    // Tell the page to show an unlocked icon. Note this is sent to the main
+    // window webcontents, not the webview webcontents
+    BrowserWindow.getAllWindows().map((win) => {
+      win.webContents.send(messages.CERT_ERROR, {
+        url: url,
+        certError: error,
+        cert: cert
```

Comment:
sorry just ignore the comma, I was just giving a nit that the value doesn't need to be specified when it is the same as the key.
So should have read:

nit: cert